src/main/resources/db/migration/V33__add_organisation_table.sql

```diff
@@ -0,0 +1,9 @@
+
+DROP TABLE organisation;
+
+CREATE TABLE IF NOT EXISTS organisation
+(
+    organisation_id UUID primary key,
+    code TEXT NOT NULL,
+    name TEXT NOT NULL
+);
```

Comment:
can we have an index on code: 


CREATE INDEX idx_organisation_code
    ON organisation(code);